pkg/occlient/occlient.go

```diff
@@ -1150,13 +1173,44 @@ func (c *Client) GetBuildConfig(name string, project string) (*buildv1.BuildConf
 // GetClusterServiceClasses queries the service service catalog to get all the
 // currently available cluster service classes
 func (c *Client) GetClusterServiceClasses() ([]scv1beta1.ClusterServiceClass, error) {
-	classList, err := c.serviceCatalogClient.ClusterServiceClasses().List(metav1.ListOptions{})
+	classList, err := c.serviceCatalogClient.ClusterServiceClasses().List(metav1.ListOptions{FieldSelector: "spec.clusterServiceBrokerName=template-service-broker"})
```

Comment:
Why is this limited to template service broker?

Reply:
as per my discussion with @containscafeine before starting this PR, we were supposed to add templates only and  Since, services from APB (ansible playbook broker) are highly parameterised, as per discussion, we are going with Template service broker only as of now (since it has default values as well). Hence you can see only templates in odo catalog list along with image names.